tket/src/Predicates/Predicates.cpp

```diff
@@ -52,9 +52,11 @@ static std::string auto_name(const T&) {
   return predicate_name(typeid(T));
 }
 
+#define SET_PRED_NAME(a) \
```

Comment:
Had to move this macro definition up here to work around a bug in the formatter: otherwise it would reformat these lines but then reformat them back again.